src/main/java/com/savvato/basemobileapp/controllers/ProfileAPIController.java

```diff
@@ -33,13 +34,17 @@ public ResponseEntity<ProfileDTO> getById(@PathVariable Long profileId) {
 	}
 	
 	@RequestMapping(value = { "/api/profile/{profileId}" }, method=RequestMethod.PUT)
-	public ResponseEntity<Boolean> update(@RequestBody @Valid ProfileRequest request) {
-		boolean rtn = profileService.update(request.userId, request.name, request.email, request.phone);
+	public ResponseEntity<GenericResponseDTO> update(@RequestBody @Valid ProfileRequest request) {
+		Boolean isUpdated = profileService.update(request.userId, request.name, request.email, request.phone);
+		GenericResponseDTO genericResponseDTO = GenericResponseDTO
```

Comment:
This works, but it makes more sense to build the dto after checking if the return value was as expected.